spannerlib/wrappers/spannerlib-dotnet/spannerlib-dotnet-grpc-impl/StreamingRows.cs

```diff
@@ -176,7 +190,9 @@ public override bool NextResultSet()
         // Read data until we reach the next result set.
         ReadUntilEnd();
         
-        return HasNextResultSet();
+        var hasNextResultSet = HasNextResultSet();
+        _pendingRow = Next();
```

Comment:
This call is needed to ensure that the error is propagated to the application if the second (or third, ...) statement in a multi-statement SQL string failed.